rasa/server.py

```diff
@@ -1455,25 +1454,15 @@ def _get_output_channel(
     )
 
 
-def _test_data_file_from_payload(
-    request: Request, temporary_directory: Path, suffix: Text = ".tmp"
-) -> Text:
-    if request.headers.get("Content-type") == YAML_CONTENT_TYPE:
-        return str(
-            _training_payload_from_yaml(
-                request,
-                temporary_directory,
-                # test stories have to prefixed with `test_`
-                file_name=f"{TEST_STORIES_FILE_PREFIX}data.yml",
-            )["training_files"]
-        )
-    else:
-        # MD test stories have to be in the `tests` directory
-        test_dir = temporary_directory / DEFAULT_CONVERSATION_TEST_PATH
-        test_dir.mkdir()
-        test_file = test_dir / f"tests{suffix}"
-        test_file.write_bytes(request.body)
-        return str(test_file)
+def _test_data_file_from_payload(request: Request, temporary_directory: Path) -> Text:
```

Comment:
https://github.com/RasaHQ/rasa/blob/7c91b14ed2241c6c3ae1f48b01cfda0ac1947718/rasa/server.py#L1067 and `_training_payload_from_json` can also be removed, right?